src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3937,7 +3937,8 @@ internal void UpdateTypes(ExecutionContext context, bool updateOnly)
 
             if (errors.Count > 0)
             {
-                var allErrors = new StringBuilder('\n');
+                // Note: don't use '\n' ('char' convert implicitly to 'ushort' and 'int' and initialize 'StringBuilder' size)
```

Comment:
This should only protect against reverting. 
(Feel free to approve the comment.)